packages/coin-sol/src/config/params.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable max-len */
+type script = {
+  [key: string]: {
+    script: string;
+    signature: string;
+    getScript: () => string;
+  };
+};
+
+export const SCRIPT: script = {
+  TRANSFER: {
+    script:
+      '03000002C70700000001F5CC071001CC071000CC071001CC071003CAA05700CAA55700CAA65700CAAC570060CC071001CC071002CC071002CC07C0020001CAAC170080CAACC7008104CAAC970085DC07C003534F4CBAA55F6C0804DDF09700250F00BAAC9ECC85080F10DAE097C009250E00D207CC05065052455353425554546F4E',
+    signature:
+      'FA0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+    getScript(): string {
+      return this.script + this.signature;
+    },
+  },
+  SMART_CONTRACT: {
+    script:
+      '03000002C70700000001F5CC071001CC071000CC071001CC071003CAA0D700FFA0CAACD70060FFE0CC071001CC071002CC071001CC071001CAAC170080CAAC970081DC07C003534F4CD207C005534D415254D207CC05065052455353425554546F4E',
+    signature:
+      'FA0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+    getScript(): string {
+      return this.script + this.signature;
+    },
+  },
+  SPL_TOKEN: {
+    script:
+      '03000002C70700000001F5CC071001CC071000CC071001CC071004CAA05700CAA55700CAA65700CAAC570060CAACD70080FFE0CC071001CC071003CC071003CC07C003020100CAAC1700A0CAAC1700A1CAAC9700A2D207CC0303534F4C53504CBAA65F6C0804DDF09700250F00BAAC9ECCA2080F10DAE097C009250E00D207CC05065052455353425554546F4E',
+    signature:
+      'FA0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+    getScript(): string {
+      return this.script + this.signature;
+    },
+  },
+};
+
+export const TRANSFER_PROGRAM_ID = Buffer.alloc(32);
```

Comment:
```suggestion
export const SYSTEM_PROGRAM_ID = Buffer.alloc(32);
```

Transfer program id is not listed in https://docs.solana.com/developing/runtime-facilities/programs. Use `SYSTEM_PROGRAM_ID` to avoid the confusion.